pwndbg/commands/heap.py

```diff
@@ -91,11 +93,37 @@ def arena(addr=None):
     main_heap   = pwndbg.heap.current
     main_arena  = main_heap.get_arena(addr)
 
-    if main_arena == None:
+    if main_arena is None:
         return
 
     print(main_arena)
 
+
+@pwndbg.commands.ParsedCommand
+@pwndbg.commands.OnlyWhenRunning
+def arenas():
+    """
+    Prints out allocated arenas 
+    """
+
+    heap  = pwndbg.heap.current
+    addr  = None
+    arena = heap.get_arena(addr)
+    main_arena_addr = int(arena.address)
+    fmt = '[%%%ds]' % (pwndbg.arch.ptrsize *2)
+    while addr != main_arena_addr:
+        
+        h = heap.get_region(addr)
+        if not h:
+            print(red('Could not find the heap'))
+            return
+        
+        print(fmt%(hex(addr) if addr else 'main'), h)
```

Comment:
this looks like a problem with printing `Page` object, works fine on my end
```
➤  b bar
Breakpoint 1 at 0x40050b
➤  r
Starting program: /tmp/a.out 

Breakpoint 1, 0x000000000040050b in bar ()
Breakpoint bar
➤  arenas
[            main]           0x602000           0x630000 rw-p    2e000 0      [heap]
```
my guess is `Page` expects `int`s not `gdb.Value` i fixed it for ptmalloc but maybe it should be changed in `Page` code?